src/components/DraggableFlatList.tsx

```diff
@@ -190,6 +191,7 @@ function DraggableFlatListInner<T>(props: DraggableFlatListProps<T>) {
       const key = keyExtractor(item, index);
       if (index !== keyToIndexRef.current.get(key))
         keyToIndexRef.current.set(key, index);
+      const passIndexToRenderItem = props.passIndexToRenderItem !== false
```

Comment:
This could use a semicolon; I think formatting with Prettier will handle this too.

I also personally think `props.passIndexToRenderItem ?? true` is a bit clearer, but either works!